libstorage/SQLStorage.cpp

```diff
@@ -107,48 +114,87 @@ Entries::Ptr SQLStorage::select(h256 hash, int64_t num, TableInfo::Ptr tableInfo
-        }
-
         Entries::Ptr entries = std::make_shared<Entries>();
-        for (Json::ArrayIndex i = 0; i < responseJson["result"]["data"].size(); ++i)
+        if (g_BCOSConfig.version() <= RC3_VERSION)
         {
-            Json::Value line = responseJson["result"]["data"].get(i, "");
-            Entry::Ptr entry = std::make_shared<Entry>();
+            std::vector<std::string> columns;
+            for (Json::ArrayIndex i = 0; i < responseJson["result"]["columns"].size(); ++i)
+            {
+                std::string fieldName = responseJson["result"]["columns"].get(i, "").asString();
+                columns.push_back(fieldName);
+            }
 
-            for (Json::ArrayIndex j = 0; j < line.size(); ++j)
+            for (Json::ArrayIndex i = 0; i < responseJson["result"]["data"].size(); ++i)
             {
-                std::string fieldValue = line.get(j, "").asString();
+                Json::Value line = responseJson["result"]["data"].get(i, "");
+                Entry::Ptr entry = std::make_shared<Entry>();
 
-                if (columns[j] == ID_FIELD)
+                for (Json::ArrayIndex j = 0; j < line.size(); ++j)
                 {
-                    entry->setID(fieldValue);
+                    std::string fieldValue = line.get(j, "").asString();
+
+                    if (columns[j] == ID_FIELD)
+                    {
+                        entry->setID(fieldValue);
+                    }
+                    else if (columns[j] == NUM_FIELD)
+                    {
+                        entry->setNum(fieldValue);
+                    }
+                    else if (columns[j] == STATUS)
+                    {
+                        entry->setStatus(fieldValue);
+                    }
+                    else
+                    {
+                        entry->setField(columns[j], fieldValue);
+                    }
                 }
-                else if (columns[j] == NUM_FIELD)
+
+                if (entry->getStatus() == 0)
                 {
-                    entry->setNum(fieldValue);
+                    entry->setDirty(false);
+                    entries->addEntry(entry);
                 }
-                else if (columns[j] == STATUS)
+            }
+        }
+        else
+        {
+            for (Json::ArrayIndex i = 0; i < responseJson["result"]["column_value"].size(); ++i)
+            {
+                Json::Value line = responseJson["result"]["column_value"][i];
+                Entry::Ptr entry = std::make_shared<Entry>();
+
+                Json::Value::Members member = line.getMemberNames();
+                for (Json::Value::Members::iterator iter = member.begin(); iter != member.end();
+                     ++iter)
                 {
-                    entry->setStatus(fieldValue);
+                    std::string key = *iter;
+                    std::string fieldValue = line.get(key, "").asString();
+                    if (key == ID_FIELD)
```

Comment:
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/623ba91f99ae7fab08971618378a74094dbd634d/libstorage/RocksDBStorage.cpp#L79-L85

can we do same as RocksDBStorage ?